cli/azd/internal/repository/infra_confirm.go

```diff
@@ -47,7 +47,7 @@ func (i *Initializer) infraSpecFromDetect(
 					DatabaseName: dbName,
 				}
 				break dbPrompt
-			case appdetect.DbPostgres:
+			case appdetect.DbPostgres, appdetect.DbMySql:
```

Comment:
Hey @rujche ! Thanks for making this change!

`infra_confirm.go` is the legacy codepath when `alpha.compose` is not enabled. Will SJAD team want to support customers in this path? For our own maintenance reasons, I think we can avoid backporting any feature additions here.

Reply:
If you are thinking about making this change, I think you'll want a different switch case to set `spec.DbMySql`